### PR TITLE
Get methods

### PR DESCRIPTION
**Pull request by stefymojica**
Hi, I have a question about md5. md5 is installed with npm?

### CODE REVIEW

File: test/GithubApi.Repositories.test.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    const response = await axios.get('https://api.github.com/repos/aperdomob/jasmine-json-report', {
+      headers: {
+        Authorization: `${process.env.ACCESS_TOKEN}`
+      }
+    });
+    const url = `${response.data.svn_url}/archive/refs/heads/master.zip`;
+    const ResponseUrl = await axios.get(url);
+    const zip = ResponseUrl.data;
+
+    // eslint-disable-next-line no-unused-expressions
```
Comment (veronicatofino) [inline test/GithubApi.Repositories.test.js:62]: Don't disable eslint

File: test/GithubApi.Repositories.test.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        headers: {
+          Authorization: `${process.env.ACCESS_TOKEN}`
+        }
+      });
+
+      expect(response.status).to.equal(StatusCodes.OK);
+      expect(response.data.description).equal('This is a Workshop about API Testing in JavaScript');
+    });
+  });
+});
```
Comment (veronicatofino) [inline test/GithubApi.Repositories.test.js:24]: This is not part of this test

File: test/GithubApi.Repositories.test.js (hunk 1 above)
Comment (stefymojica) [reply test/GithubApi.Repositories.test.js:62]: I have to disable the rule to avoid the error "Expected an assignment or function call and instead saw an expression." because the to.exist is not a function and can not be used as to.exist()

File: test/GithubApi.Repositories.test.js (hunk 2 above)
Comment (stefymojica) [reply test/GithubApi.Repositories.test.js]: Fixed.,Thanks.

File: package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "description": "This is a Workshop about Api Testing in JavaScript",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint && mocha -t 5000 --reporter mochawesome --reporter-options reportDir=report,reportFilename=ApiTesting",
+    "test": "yarn lint && mocha -t 5000 --reporter mochawesome --reporter-options reportDir=report,reportFilename=ApiTesting",
```
Comment (kliver98) [inline package.json:7]: why did you change to yarn? I suggest to keep using npm

File: test/GithubApi.Repositories.test.js (hunk 1 above)
Comment (kliver98) [reply test/GithubApi.Repositories.test.js:42]: You can read [chai documentation](https://www.chaijs.com/api/bdd/#method_above) and see any other possible solution.
For instance you could do `expect(zip).to.have.lengthOf.above(1);` because we know zip should be an string and should have some content (greater than 1) and will not needed to disable eslint

Comment (kliver98) [reply test/GithubApi.Repositories.test.js:42]: You can read [chai documentation](https://www.chaijs.com/api/bdd/#method_above) and see any other possible solution.
For instance you could do `expect(zip).to.have.lengthOf.above(1);` because we know zip should be an string and should have some content (greater than 1) and will not needed to disable eslint
@veronicatofino @stefymojica

Comment (stefymojica) [reply test/GithubApi.Repositories.test.js:42]: Great, thanks.

**Review by holgiosalos — APPROVED** [review]
